extensions/interactions/AlgebraicExpressionInput/protractor.js

```diff
@@ -17,10 +17,29 @@
  * interaction
  */
 
+var action = require(process.cwd() + '/core/tests/protractor_utils/action.js');
 var objects = require(process.cwd() + '/extensions/objects/protractor.js');
+var waitFor = require(
+  process.cwd() + '/core/tests/protractor_utils/waitFor.js');
 
-var customizeInteraction = function() {
-  // There are no customizations.
+var customizeInteraction = async function(elem, customLetters) {
+  await waitFor.presenceOf(elem.element(by.css(
+    '.protractor-test-custom-letters-div')), 'Element took too long to load.');
```

Comment:
Can you make this error message more specific? In particular, describe which element took long to load. This is important because we don't always get reliable line numbers in stack traces, so the message has to be enough to debug any problems

Reply:
Done.